package.json

```diff
@@ -40,7 +40,7 @@
     "bytes": "^2.4.0",
     "classnames": "^2.2.5",
     "co-body": "^4.2.0",
-    "config": "^1.25.1",
+    "@steem/config": "^1.25.1",
```

Comment:
@valzav i think this should remain the same - it's causing 'yarn' to fail to install/build.